cpanfile

```diff
@@ -1,5 +1,5 @@
 requires 'DBI';
-requires 'DBD::mysql';
+requires 'DBD::mysql', '<= 4.050'; # newer versions do not support MySQL 5
```

Comment:
We should use 4.051 here.
With that OK from me.

Reply:
Changed to comply and accommodate further possible minor updates.